packages/video_player/video_player_android/android/src/main/java/io/flutter/plugins/videoplayer/VideoPlayer.java

```diff
@@ -110,7 +111,7 @@ final class VideoPlayer {
   }
 
   @VisibleForTesting
-  public void buildHttpDataSourceFactory(@NonNull Map<String, String> httpHeaders) {
+  public void setupHttpDataSourceFactory(@NonNull Map<String, String> httpHeaders) {
```

Comment:
Nit: "setup" is a noun, the verb is "set up", so this should be "setUpHttpDataSourceFactory".

Reply:
oh, well, this now looks like a settter?
chosed configure if you don't mind